### ISSUE TITLE
Mapping from pointing_button not having an effect

### ISSUE THREAD

**Issue by dylan-chong** [tekezo/Karabiner-Elements#1192]
The below complex mapping has no effect.

There is no error in the log.

```
                        "manipulators": [
                            {
                                "type": "basic",
                                "from": {
                                    "pointing_button": "button6" # also does not work with button2
                                    # works with  "key_code": "left_arrow" instead
                                },
                                "to": [
                                    {
                                        "key_code": "left_arrow",
                                        "modifiers": [
                                            "left_control",
                                            "left_shift"
                                        ]
                                    }
                                ]
                            }
                        ]
```

**Comment by dylan-chong** [comment]
Interestingly this mapping does work:

```
                            {
                                "type": "basic",
                                "from": {
                                    "key_code": "z"
                                },
                                "to": [
                                    {
                                        "pointing_button": "button2"
                                    }
                                ]
                            },
```

**Comment by tekezo** [comment]
Did you enable your mouse?
https://pqrs.org/osx/karabiner/help.html#mouse-button

**Comment by dylan-chong** [comment]
I did not! Thanks!

**Comment by eret9616** [comment]
> Did you enable your mouse?
> https://pqrs.org/osx/karabiner/help.html#mouse-button

this is isn't avaliable any more

**Comment by space-chicken** [comment]
Since the link isn't available anymore,

> Karabiner-Elements does not modify mouse devices by default.
> You have to enable your mouse on `Karabiner-Elements Preferences > Devices`

**Comment by momomo** [comment]
Same here. No effect.

**Comment by momomo** [comment]
This helped: 

https://karabiner-elements.pqrs.org/docs/help/how-to/mouse-button/

**Comment by fourcolors** [comment]
Thank you! This was the problem for me too.

**Comment by fourcolors** [comment]
TL;DR; 

1. Enable mouse events for your device 

<img width="1193" alt="Image" src="https://github.com/user-attachments/assets/a25a9196-58ff-4de0-881f-c0c2656eb3c5" />

2. Write custom events and profit (this now works)

<img width="1028" alt="Image" src="https://github.com/user-attachments/assets/bdc8ccb4-f179-42cc-8236-9d7499704e11" />
